src/MaterialUI.re

```diff
@@ -1471,6 +1482,91 @@ module Table = {
     );
 };
 
+module TextField = {
+  [@bs.module "material-ui/TextField"] external reactClass : ReasonReact.reactClass = "default";
+  let make =
+      (
+        ~autoComplete: option(string)=?,
+        ~autoFocus: option(bool)=?,
+        ~className: option(string)=?,
+        ~defaultValue: option(string)=?,
+        ~disabled: option(bool)=?,
+        ~error: option(bool)=?,
+        ~formHelperTextProps: option(Js.t({..}))=?,
+        ~fullWidth: option(bool)=?,
+        /* TODO: change to more appropriate type*/
+        ~helperText: option(string)=?,
+        ~helperTextClassName: option(string)=?,
+        ~id: option(string)=?,
+        /* Material-UI have two pais of similar props with different case. See: Git hub issue https://github.com/callemall/material-ui/issues/8232 */
+        ~inputClassName: option(string)=?,
+        ~_InputClassName: option(string)=?,
+        ~_InputLabelProps: option(Js.t({..}))=?,
+        ~inputProps: option(Js.t({..}))=?,
+        ~_InputProps: option(Js.t({..}))=?,
+        /* TODO: Find Function type in ReasonMl */
+        ~inputRef=?,
```

Comment:
Change todo to `/* TODO: change type to match ref callback (see https://reasonml.github.io/reason-react/docs/en/react-ref.html) */`